include/swift/Sema/CSFix.h

```diff
@@ -2196,6 +2209,80 @@ class SpecifyBaseTypeForOptionalUnresolvedMember final : public ConstraintFix {
           MemberLookupResult result, ConstraintLocator *locator);
 };
 
+class CheckedCastContextualMismatchWarningBase : public ContextualMismatch {
```

Comment:
Let's drop `Base` since it's already an abstract class.

Reply:
Fixed, thanks :)